content/utilities/foundries-score/plugin.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * Foundries Score
+ * 
+ * Summary points of your unprospected foundries
+ * 
+ * Level 1 = Common - [100%] - [5k points]
+ * Level 2 = Rare - [1/64(1.56%) for Epic] - [~ 23k points]
+ * Level 3 = Rare - [1/16(6.25%) for Epic] - [~ 31k points]
+ * Level 4 = Epic - [1/64(1.56%) for Legendary] - [~ 244k points]
+ * Level 5 = Epic - [1/16(6.25%) for Legendary] - [~ 375k points]
+ * Level 6 = Legendary - [1/64(1.56%) for Mythic] - [~ 3.3m points]
+ * Level 7 = Legendary - [1/16(6.25%) for Mythic] - [~ 4.1m points]
+ * Level 8+ = Mythic - [100%] - [20m points]
+ * 
+ * Common - 5k points
+ * Rare - 20k points
+ * Epic - 200k points
+ * Legendary - 3M points
+ * Mythic - 20M points
+ * 
+*/
+
+const planetLevels = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+const colors = ['#5c5c5c', '#5c5c5c', '#5c5c5c', '#a0a0a0', '#00dc80', '#6b68ff', '#c13cff', '#f8b73e', '#ff44b7'];
+const points = [0, 5000, 23000, 31000, 244000, 375000, 3300000, 4100000, 20000000, 20000000];
+
+
+const players = [
+    df.getAccount(),
+    // Crawls nearby unowned spacetimes
+    "0x0000000000000000000000000000000000000000",
+];
+
+class Plugin {
+    constructor() {
+        this.content = document.createElement('div');
+        this.total = document.createElement('div');
+        this.message = document.createElement('div');
+        this.refreshInterval = 1; // minutes
+    }
+
+    render(container) {
+        // Create refresh interval
+        this.refreshTimer = setInterval(() => {
+            setTimeout(() => {
```

Comment:
Curious on why this `setTimout` is needed here?

Reply:
Indeed?

Reply:
It's not needed in this place, already removed it.

I'm using this method when I need to trigger some method only after DOM re-render, like here:
https://github.com/darkforest-eth/plugins/pull/173/files#diff-3c980389c28dd9c062a5a9ff0780bece3dd742b6ed985431965f7fa01e232e43R71

Maybe not the best way to do it.